tests/modules/s3_datasets_shares/test_share.py

```diff
@@ -2682,7 +2682,7 @@ def test_delete_share_object_remaining_items_error(
         client=client, user=user2, group=group2, shareUri=share3_processed.shareUri
     )
     # Then we get an error of the type
-    assert 'UnauthorizedOperation' in delete_share_object_response.errors[0].message
+    assert 'ShareItemsFound' in delete_share_object_response.errors[0].message
```

Comment:
I guess this is fixing an existing bug (we return the wrong message), right?

Reply:
Yes, this was a small bug